server/server.go

```diff
@@ -883,6 +882,14 @@ func (m *Command) Close() error {
 
 		err := eg.Wait()
 		_ = testhook.Closed(pilosa.NewAuditor(), m, nil)
+		if m.Config.DataDog.Enable {
+			defer profiler.Stop()
+		}
+		if m.traceCloser != nil {
+			defer m.traceCloser.Close()
```

Comment:
why defer?

Reply:
probably doesn't matter, but I figured it was best to stop the profiler when the Close actually completed... in case some part of the Close ended up hanging or something.